Replicator/DBAccess.cc

```diff
@@ -245,13 +253,24 @@ namespace litecore { namespace repl {
             string attName = string("blob_") + string(path);
             enc.writeKey(slice(attName));
             enc.beginDict();
+            bool cbl_4944_findLength = false;
             for (Dict::iterator i(blob); i; ++i) {
                 slice key = i.keyString();
+                if (key == "length"_sl) {
+                    cbl_4944_findLength = true;
```

Comment:
CBSE case has attachment length = 0 while it shouldn't be 0. The attachment length = 0 could be valid for empty blob but for the purpose of the investigation, we should log as well.